### PR TITLE
CLI-Command with from and to date parameters

### PR DESCRIPTION
**Pull request by brxbrx**
Now you can do sth like this:

php bin/console sw:importexport:export -p "MyProfile" --dateFrom "2018-11-01" --dateTo "2018-11-31" Export.csv -f csv

### CODE REVIEW

File: Commands/SwagImportExport/ExportCommand.php [hunk 1]
```diff
@@ -161,6 +184,31 @@ protected function prepareExportInputValidation(InputInterface $input)
...
         $this->category = $input->getOption('category');
         $this->productStream = $input->getOption('productStream');
+        $this->dateFrom = $input->getOption('dateFrom');
+        $this->dateTo = $input->getOption('dateTo');
+
+        if (!empty($this->dateFrom)) {
+            try {
+                $this->dateFrom = new \DateTime($this->dateFrom);
+            } catch (Exception $e) {
+                sprintf('Invalid format! %s', $e->getMessage());
```
[the remark this answers is not in the document]
Comment (brxbrx) [reply Commands/SwagImportExport/ExportCommand.php]: Yes, you're right.

File: Commands/SwagImportExport/ExportCommand.php [hunk 2]
```diff
@@ -161,6 +184,31 @@ protected function prepareExportInputValidation(InputInterface $input)
...
+            }
+        }
+
+        if (!empty($this->dateTo)) {
+            try {
+                $this->dateTo = new \DateTime($this->dateTo);
+                $this->dateTo->setTime(23, 59, 59);
+            } catch (Exception $e) {
+                sprintf('Invalid format! %s', $e->getMessage());
```
[the remark this answers is not in the document]
Comment (brxbrx) [reply Commands/SwagImportExport/ExportCommand.php]: yes, see above.

**Comment by brxbrx** [comment]
fixed

**Comment by brxbrx** [comment]
fixed

**Comment by htkassner** [comment]
Thank you for your contribution. This was merged and will be released in one of the next releases.

**Pull request closed without merging**